.github/issue_template.md

```diff
@@ -0,0 +1,41 @@
+#### A note about questions vs. issues
+
+Please understand that **this is not a forum** but an issue tracker.
+
+The following article explains why you must not ask questions on Github issues: https://medium.com/@methane/why-you-must-not-ask-questions-on-github-issues-51d741d83fde
+
+Invalid issues will be closed without any comment.
+
+
+#### Brief description
+
+< describe the main issue in one sentence >
+
+
+#### Affected components / protocols
+
+< list what is affected by the issue (e.g. wrpcap + IPv6,it is likely this also affects XXX) >
```

Comment:
I think this part won't be used, and could be merged with the one above.
Correct me If I'm wrong

Reply:
Agree

Reply:
see 74e4a7e